### PR TITLE
Adds tooltips to all flock structures

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[QOL]
## About the PR 
Hovering over a flock structure in the TGUI control panel now brings up a tooltip with a brief description of what it does.
image

## Why's this needed? 
Suggested during playtest, players should be able to see what things do.

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
There should be a tooltip in the construct structures tab too

File: tgui/packages/tgui/interfaces/FlockPanel.js [hunk 1]
```diff
@@ -201,12 +201,14 @@ const FlockStructures = (props, context) => {
...
-                  <Stack vertical align="center">
-                    <Stack.Item >{structure.name}</Stack.Item>
-                    <Stack.Item >{structure.health}<Icon name="heart" /></Stack.Item>
-                  </Stack>
-                </Section>
+                <Tooltip position="bottom" content={structure.desc}>
+                  <Section position="relative">
+                    <Stack vertical align="center">
+                      <Stack.Item >{structure.name}</Stack.Item>
+                      <Stack.Item >{structure.health}<Icon name="heart" /></Stack.Item>
```
Comment (ZeWaka) [inline tgui/packages/tgui/interfaces/FlockPanel.js:208]:
```suggestion
                      <Stack.Item >{structure.health} <Icon name="heart" /></Stack.Item>
```
ugly

Comment (ZeWaka) [reply tgui/packages/tgui/interfaces/FlockPanel.js:208]: will need rebuild

File: code/obj/flock/structure/compute.dm [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /obj/flock_structure/compute
 	name = "weird lookin' thinking thing"
 	desc = "It almost looks like a terminal of some kind."
+	flock_desc = "The remains of a human computer, adapted to provide compute power to the Flock."
```
Comment (FlameArrow57) [inline code/obj/flock/structure/compute.dm:4]:
```suggestion
	flock_desc = "A computing node that provides compute power to the Flock."
```
Or similar? Feel like mentioning human computer isn't much necessary, for conciseness/relevancy reasons (already been converted), but either probably fine

File: code/obj/flock/structure/rift.dm [hunk 3]
```diff
@@ -7,6 +7,7 @@
 	density = FALSE
 	name = "glowing portal thingymabob"
 	desc = "Oh god is that a fucking light grenade?!"
+	flock_desc = "The rift through which your Flock will enter this world."
```
Comment (FlameArrow57) [inline code/obj/flock/structure/rift.dm:10]:
```suggestion
	flock_desc = "The rift through which your Flock will begin its journey."
```
Or similar? Sounds cool, but maybe feels a bit out of theme, idk

**Comment by FlameArrow57** [comment]
Minor thing, not much necessary, but some descriptions were actual sentences whereas other ones weren't, maybe could stick with one or other for all? Again, minor

File: code/obj/flock/cage.dm [hunk 4]
```diff
@@ -4,6 +4,7 @@
 /obj/flock_structure/cage
 	name = "weird energy cage"
 	desc = "You can see the person inside being rapidly taken apart by fibrous mechanisms. You ought to do something about that."
+	flock_desc = "Spins living matter into Flockdrones. Painfully."
```
[the remark this answers is not in the document]
Comment (TobleroneSwordfish) [reply code/obj/flock/cage.dm:7]: Mob is kind of a meta term, I'm basing this off the ghostdrone definition of living which includes silicons.

File: code/obj/flock/structure/egg.dm [hunk 5]
```diff
@@ -8,6 +8,7 @@
 	density = FALSE
 	name = "glowing doodad"
 	desc = "Oh god is that a fucking light grenade?!"
+	flock_desc = "Will soon become a Flockdrone."
```
[the remark this answers is not in the document]
Comment (TobleroneSwordfish) [reply code/obj/flock/structure/egg.dm:11]: Well create implies it might hang around afterwards?

File: code/obj/flock/structure/rift.dm (hunk 3 above)
Comment (TobleroneSwordfish) [reply code/obj/flock/structure/rift.dm:10]: 🤷‍♀️

File: code/obj/flock/structure/egg.dm (hunk 5 above)
[the remark this answers is not in the document]
Comment (FlameArrow57) [reply code/obj/flock/structure/egg.dm:11]: Hatch?

File: tgui/packages/tgui/interfaces/FlockPanel.js (hunk 1 above)
Comment (TobleroneSwordfish) [reply tgui/packages/tgui/interfaces/FlockPanel.js]: ?

**Pull request merged**